### DISCUSSION TITLE
Migration assistance (visually impaired)

### DISCUSSION THREAD

**Discussion by b9AcE** [martinrotter/rssguard#1440] [Q&A]
I have been using QuiteRSS for probably on average >10 hours almost every day for over a decade but since it is apparently long dead I must find an alternative and I am hoping RSS Guard is a usable option for me.
I have been searching RSS Guard's settings and menus, but I (very visually impaired, white cane, full disability) can't find how or if it is possible, so I thought I'd just ask in a place where people are likely to know, here:

I want to be able to go through all articles globally all, per feed all or per feed tree-category all, marking those that need later action as "important" (or similar) so that those will absolutely definitely not get deleted until unmarked again, while deleting each article determined to need no (further) action so that when I am all caught up globally or per feed/category I would have no articles displayed in the list except those marked as important and therefore won't be removable until they are no longer important, so then I would be able to tell RSS Guard to "delete all" every time that it has fetched new articles and I have seen that none of those new require (further) action and that would delete all articles EXCEPT NOT the ones marked as important of course, then just unmark as important once an article has been sufficiently acted upon making it no longer important and therefore can be deleted too, etc, etc.

As far as I can see (again, very visually impaired) the "mark as important" does not make the articles treated as particularly important at all, but just adds some faint icon, but we are still able to delete important articles? That seems like it should not be desired by anyone, so I am wondering if I am missing something, did I accidentally enable some "enable deleting important override" setting somewhere that I can not find to revert?

Also, is there an option somewhere to have a keyboard shortcut actually put articles only in the recycle bin (except not important ones) and not show them in the main list, instead of just crossing them out?
Since I became very visually impaired, it is impacting my fatigue insurmountably much to keep having articles listed that I have to strain myself a lot to repeatedly check the same ones over and over again just to spot the to me barely even visible overstrike (while zoomed in 150% using font size 28 on a 55 inch screen) marking it as apparently placed in recycle bin but bafflingly still also outside the recycle bin,
so I would appreciate very much if someone here point me to how I can use RSS Guard in a way accessible to me with this condition.
Otherwise, I have found no other viable candidates, so I seem to be stuck using deprecated QuiteRSS forever. :-/

Thanks!

**Comment by Ac314** [comment]
The application has capabilities to automatically clean up the database from the old articles. And this option has the modifiers "Do not remove important articles" and "Do not remove unread articles". So if you rely on automatic cleaning you could set it to never delete important and/or unread articles so you will never miss them.
Unfortunately I cannot say much about manual deleting procedure because I never did it. Automatic cleaning works quite well for me - read the new article, mark it important if you want to save it for the future, or, if you and do not need it, just forget about it, it will be deleted automatically after some time.
I understand that maybe you used to some other usage scenario but automatic cleaning could be useful too. Less worry about the old messages, the program will do it for you.

**Reply by b9AcE** [reply]
Thank you very much for the reply! :-)

If that is the actual current limitation of RSS Guard, that would be very unfortunate as it would make it impossible for me to use as replacement for QuiteRSS with my visual impairment.
Not a matter of preference, but practically speaking impossible, determined by prolonged test attempting to use RSS similarly to as you described daily quickly resulting in debilitating exhaustion.

If anyone knows of a solution to this issue whether by capabilities in RSS Guard or elsewhere, suggestions would be extremely welcome.

**Reply by Ac314** [reply]
I will try to describe how I personally treat old articles (including important ones). I understand that the way I use it could be inconvenient for you but maybe you could find some useful ideas...

First, article list could be filtered to hide articles you do not need. In my case articles I do not need are already read articles which I did not mark "important". So my default filter is "Show unread articles" and "Show important articles".

When I have the new articles I read them when I have time, mark some of them important if I want to save them and go on with reading some other feed. When I return to this feed again I see only still unread messages and important messages. Read messages are hidden from the view and this is good because I read them already and, because I did not mark them important, I do not need them.

If I want to return to old messages I can temporarily switch the article list filter to "No extra filtering" (meaning "Show all messages"). Then I will see all messages in the feed.

Most convenient way to enable article list filters is to have a button "Article list filter" on toolbar for articles list. Then you can set/change the filters fast when you need it.

Second, with regards to deleting the old articles - there is no need to manually delete them. Cleaning the database (manually or automatically) functionality is more convenient because you can delete all not needed messages at once. 

I personally use automatic cleaning meaning I only need to make the right settings for my feeds and then do not care about the old messages. In my case only read messages which are not marked "important" are deleted so there is no need to worry about losing something important. 

But if for some reasons you do not like the automatic cleaning the manual database cleaning can also be used (Tools -> Cleanup database). You can run it from time to time and it will delete old articles depending on your settings (like "delete articles older than 14 days"). You can also set the option "Do not delete important messages", and you will not lose them while cleaning.  

So if you make database cleaning you do not need to manually delete the articles one by one and use Recycle Bin for that. You can just leave them be, they will be deleted during database cleaning. And enabling the article list filters will allow you to hide them from viewing.

**Reply by b9AcE** [reply]
Thanks again for the reply.

I am already using "mark as read" for other essential purposes, so it can not be used for other purposes.

I believe I can take it as implied from your responses and that nobody else has said anything to the contrary, that the "mark as important" is incapable of doing anything except set a visual flag, which I can't see during normal use anyway, so effectively nothing in my case.
That is extremely unfortunate, as having important articles treated as too important too delete simply is the absolutely essential to my being able to handle the large amounts of incoming articles every day especially since becoming visually impaired.

It seems I will have to keep looking and hoping some other RSS-client is designed more with impaired people in mind.

Thanks for trying though, @Ac314. :-)
